configure

```diff
@@ -124,14 +125,14 @@ while :; do
 				JSON
 			)
 			if echo "$response" | grep '"token"'; then
-				echo "$response" | jq -r .token | read i_api_token
+				echo "$response" | jq -r .token | read i_access_token
 			else
 				printf "\033[31;31mError: \033[0m"
 				echo "$response" | jq -r .message
 				echo "You'll need to create a GitHub API token manually."
 			fi
 
-			[ -n "$i_api_token" ] && break
+			[ -n "$i_access_token" ] && access_token="$i_access_token" && break
```

Comment:
Great! Just one more thing — I find >2 statements on a single line to be difficult to read. Could we change this to an `if` with the variable assignment and the `break` each on their own lines? Thanks!